common-components/proto/aip/type/money.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package aip.type;
+
+option cc_enable_arenas = true;
+option go_package = "aip.golang.org/genproto/common-components/type/money;money";
+option java_multiple_files = true;
+option java_outer_classname = "MoneyProto";
+option java_package = "dev.aip.type";
+option objc_class_prefix = "AIP";
+
+import "aip/type/decimal.proto";
+
+// Represents an amount of money with its currency type.
+//
+// Examples:
+//
+// Five US dollars === `{currency_code: "USD" quantity: {significand: 5}}`
+// One and a half Bitcoin ===
+//   `{currency_code: "X-BTC" quantity: {significand: 15 exponent: -1}}`
```

Comment:
```suggestion
// Five US dollars === `{"currency_code": "USD", "quantity": {"significand": 5}}`
// One and a half Bitcoin ===
//   `{"currency_code": "X-BTC", "quantity": {"significand": 15, "exponent": -1}}`
```

Reply:
I don't think this suggestion is right, because these examples are written as text format protos.